include/multipass/virtual_machine.h

```diff
@@ -112,6 +112,7 @@ class VirtualMachine : private DisabledCopyMove
     virtual void load_snapshots() = 0;
     virtual std::vector<std::string> get_childrens_names(const Snapshot* parent) const = 0;
     virtual int get_snapshot_count() const = 0;
+    virtual void remove_snapshots_from_image() const = 0;
```

Comment:
I think the snapshot data has different parts. 1. the snapshot file. 2. the RAM part, meaning the c++ snapshot object. 3. the data in the image. 
What we are deleting here is only the 3rd one, right? We also want to avoid names which cause ambiguity.

Reply:
Done